virt/virt-4-8-release-notes.adoc

```diff
@@ -145,6 +145,47 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-8-known-issues"]
 == Known issues
 
+* If you initially deployed {VirtProductName} at version 2.4.z or earlier, upgrading to version 4.8 fails with the following message:
++
+[source,terminal]
+----
+risk of data loss updating hyperconvergeds.hco.kubevirt.io: new CRD removes version v1alpha1 that is listed as a stored version on the existing CRD
+----
++
+This bug does not affect clusters where {VirtProductName} was initially deployed at version 2.5.0 or later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986989[*BZ#1986989*])
++
+** As a workaround, remove the `v1alpha1` version and resume the upgrade process.
++
+. Run a proxy to the Kubernetes API server by running the following command:
++
+[source,terminal]
+----
+$ oc proxy &
+----
++
+. Remove the `v1alpha1` version from `.status.storedVersions` on the `HyperConverged` custom resource definition (CRD) by running the following command:
++
+[source,terminal]
+----
+$ curl --header "Content-Type: application/json-patch+json" --request PATCH http://localhost:8001/apis/apiextensions.k8s.io/v1/customresourcedefinitions/hyperconvergeds.hco.kubevirt.io/status --data '[{"op": "replace", "path": "/status/storedVersions", "value":["v1beta1"]}]'
+----
++
+. Resume the upgrade process by running the following command:
++
+[source,terminal]
+----
+$ curl --header "Content-Type: application/json-patch+json" --request PATCH http://localhost:8001/apis/operators.coreos.com/v1alpha1/namespaces/openshift-cnv/installplans/$(oc get installplan -n openshift-cnv | grep kubevirt-hyperconverged-operator.v4.8.0 | cut -d' ' -f1)/status --data '[{"op": "remove", "path": "/status/conditions"},{"op": "remove", "path": "/status/message"},{"op": "replace", "path": "/status/phase", "value": "Installing"}]'
+----
++
+. Kill the `oc proxy` process. For example:
```

Comment:
I am not sure you need "For example". This command will kill the proxy process. There are lots of ways to kill processes and this command is as good as any..